Index: leptonica-1.68/src/pngio.c
===================================================================
--- leptonica-1.68.orig/src/pngio.c
+++ leptonica-1.68/src/pngio.c
@@ -107,7 +107,7 @@
 /* --------------------------------------------*/
 
 #include "png.h"
-
+#include "zlib.h"
 /* ----------------Set defaults for read/write options ----------------- */
     /* strip 16 bpp --> 8 bpp on reading png; default is for stripping */
 static l_int32   var_PNG_STRIP_16_TO_8 = 1;
